src/scss/components/events-highlights.scss

```diff
@@ -0,0 +1,83 @@
+.event {
+  $root: &;
+  @extend .container;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 80px;
+
+  &.feature {
+    padding-bottom: 100px;
+    #{$root}__title {
+      font-size: $mobile-title;
+      order: -1;
+    }
+  }
+
+  > * {
+    margin: 0 auto;
+    max-width: 425px;
+    padding: 0 19px;
+    width: 100%;
+  }
+
+  &__title {
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-card-titles;
+    padding-bottom: 20px;
+  }
+
+  &__image {
+    padding-bottom: 20px;
+  }
+
+  &__text {
+    color: $gray;
+    font-family: $hkgrotesk-regular;
+    font-size: $mobile-paragraph;
+    padding-bottom: 40px;
+  }
+
+  &__more-info {
+    .more-info {
+      display: flex;
+      flex-direction: column;
+
+      &__text {
+        font-family: $hkgrotesk-bold;
+        font-size: $mobile-card-titles;
+        width: 56px;
+      }
+
+      &__description {
+        color: $gray;
+        font-family: $hkgrotesk-regular;
+        font-size: $mobile-paragraph;
+        margin-left: 12px;
+      }
+
+      &__wraper {
+        display: flex;
+        padding-bottom: 20px;
+      }
+    }
+  }
+
+  &__link {
+    background-color: $green;
+    display: block;
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-card-titles;
+    margin: 0 auto;
+    max-width: 100%;
+    text-align: center;
+    width: 318px;
+    a {
+      color: $black;
+      display: block;
+      height: 60px;
```

Comment:
If you need to set a `height`, do it in the container of this element because by default the `block` elements will inherit the dimensions of their parent